src/action.js

```diff
@@ -6,59 +6,62 @@ const registerDependencyAndroid = require('./android/registerNativeModule');
 const registerDependencyIOS = require('./ios/registerNativeModule');
 const copyAssetsAndroid = require('./android/copyAssets');
 const copyAssetsIOS = require('./ios/copyAssets');
+const pjson = require(path.join(process.cwd(), './package.json'));
```

Comment:
To be more explicit, you can add that require above line 37 so we do not require from inside a function. On the other hand, we do not need that `pjson` anywhere in the action, so might be good leaving it inside that function.